modules/win_mozilla_build_tester/manifests/modifications.pp

```diff
@@ -3,12 +3,11 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 class win_mozilla_build_tester::modifications {
-
   require win_mozilla_build_tester::install
 
   $mozbld      = "C:\\mozilla-build"
 
-  file { "${mozbld}\\python\\Scripts\\hg":
+  file { "${mozbld}\\python3\\Scripts\\hg":
```

Comment:
Will this be OK on the Azure testers as well?

Reply:
@markcor Yes, this PR was pulled from an upstream `win11` branch that didn't have that change that `cloud_windows` has. Link [here](https://github.com/mozilla-platform-ops/ronin_puppet/blob/cloud_windows/modules/win_mozilla_build_tester/manifests/modifications.pp#L10).